code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -295,8 +295,11 @@
 	if(ishuman(usr))
 		add_fingerprint(usr)
 		toggle(usr)
-	else
-		to_chat(usr, "<span class='warning'>This mob type can't use this verb.</span>")
+		return
+	if(isrobot(usr))
+		toggle(usr)
+		return
```

Comment:
```suggestion
```

You can get rid of this and add the check in the above `if` block. `add_fingerprint` is needed for hidden fingerprints for admins.